src/test/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSparkUnitTest.java

```diff
@@ -22,6 +22,21 @@
 public final class FindBreakpointEvidenceSparkUnitTest extends BaseTest {
     private static final SVInterval[] testIntervals =
             { new SVInterval(1, 33140934, 33141485), new SVInterval(1, 33143116, 33143539) };
+    private static final SVInterval[] testBigIntervals = {
```

Comment:
Rather than put these into two separate arrays, you probably want to use an `@DataProvider` to feed individual inputs to the method along with their expected results. You can find documentation for that in the testng package. It'll make your test a little more compact, and you won't have to do the stream in the test method to collect the results.